src/main/java/com/mxf/rest/entity/Article.java

```diff
@@ -0,0 +1,92 @@
+package com.mxf.rest.entity;
+
+import java.sql.Date;
+
+public class Article extends ABaseEntity {
+	
+	private int id;
+	private String title;
+	private String label;
+	private String text;
+	private Date date;
+	private Integer authorId;
+	
+	public Article() {
+		this.title = "";
```

Comment:
really they have to be equals empty? What reason of it? Do you have some specific logic or magic ritual to create article?